apps/prairielearn/src/lib/client/page-context.test.ts

```diff
@@ -0,0 +1,143 @@
+import { describe, expect, it } from 'vitest';
+
+import {
+  type StaffCourseInstanceContext,
+  type StudentCourseInstanceContext,
+  getCourseInstanceContext,
+  getPageContext,
+} from './page-context.js';
+
+describe('getPageContext', () => {
+  it('strips extra fields from the data', () => {
+    const mockData = {
+      authz_data: {
+        has_course_instance_permission_edit: true,
+        has_course_instance_permission_view: true,
+        has_course_permission_own: true,
+        user: { name: 'Test User', uid: 'test@illinois.edu' },
```

Comment:
Can we add extra properties to `authz_data.user` and `authn_user` to assert that they're stripped?